crates/storage/src/schema.rs

```diff
@@ -1,14 +1,6 @@
 mod base;
 
-mod revision_0031;
-mod revision_0032;
-mod revision_0033;
-mod revision_0034;
-mod revision_0035;
-mod revision_0036;
-mod revision_0037;
-mod revision_0038;
-mod revision_0039;
+mod revision_0040;
```

Comment:
Just making sure I get it right: we won't need these previous revisions because a full sync from scratch is required, correct?

Reply:
Yes the idea is basically that the new 'base' version is now the aggregate of all 39 original migrations. And then 0040 is added on as the "first" real migration now.

We've done this previously but up to 31 - which is why the first 30 migrations were also missing. This was done because those migrations contained a bunch of coupling to the rest of the codebase; and anyone migrating from such an old db should really just resync :)

This time around its for a different reason - there just isn't a pragmatic way to migrate this much data.